front-end-master/landing_page/index.css

```diff
@@ -4,7 +4,7 @@ html,body
     height: 100%;
     margin: 0px;
     padding: 0px;
-    overflow-x: hidden;
+    /overflow-x: hidden;
```

Comment:
Did you mean to leave this here?

Reply:
yes kind of. the thing is scrollEvent never fires up if the overflow is hidden. but if i don't keep the overflow-x hidden sometimes a scrollbar appears at the bottom of the page the document gets a little wider than the viewpoint. though it is not likely to happen but i added the overflow-x hidden anyway for being on the safe side. as i need the scroll event to occur i had to comment it. is the any way to fire scroll even when i have a overflow-x hidden? i just need the scroll event for y axis and overflow x was not supposed to make a any problem here but it does :(

Reply:
Well, I guess remove the comment syntax then?